lib/parallel_tests/cli.rb

```diff
@@ -192,6 +192,12 @@ def parse_options!(argv)
           options[:isolate] = true
         end
 
+        opts.on("--isolate-n [PROCESSES]",
+          Integer,
+          "Number of processes to use for isolated singles, default: 1. Do not set this without setting --isolate.") do |n|
```

Comment:
how about making it turn isolate on too ... then we don't need the extra test/raise etc ?

Reply:
also test here should match the readme can run `--help` and then paste to the readme

Reply:
Great idea.